docs/intro/validator.rst

```diff
@@ -0,0 +1,225 @@
+=========
+Validator
+=========
+
+Getting started
+===============
+Validator is an instrument to unify the process of model training and validation.
+There are two main reasons to wrap your model using classes from `Validator` submodule:
+
+- make the structure of the model lifecycle clear,
+- provide API to automized model training and validation.
+
+All you need is:
+
+- define class inherited from `Validator`,
+- configure it by `yaml` file,
+- execute the following code: ::
```

Comment:
Let's add a default script which runs validation process

Reply:
I have such a script in `ml_project_template`. Do we really need it in `batchflow` if `validator` is just an interface?